playbooks/robusta_playbooks/pod_troubleshooting.py

```diff
@@ -268,6 +279,10 @@ def debugger_stack_trace(event: PodEvent, params: DebuggerParams):
     if not pid:
         logging.info(f"debugger_stack_trace - no relevant pids")
 
+    if params.traces_amount < 1 or params.sleep_duration_s < 0:
```

Comment:
I think min duration should be 1
Regardless, if this is an error, then shouldn't we return here?

Reply:
default sleep duration is 1 but its possible to do repetitive stack traces with no sleep